library/src/main/java/com/schibsted/spain/barista/BaristaRadioButtonActions.java

```diff
@@ -13,17 +14,20 @@
 public class BaristaRadioButtonActions {
 
   public static void clickRadioButtonItem(@IdRes int radioGroupId, @IdRes int itemToClickId) {
+    scrollTo(radioGroupId);
```

Comment:
Actually, `Barista.scrollTo()` does the trick of doing the scroll 100 times to fix an unknown Espresso bug. And I guess that just perming an Espresso's scrollTo() doesn't work on some scenarios. But great question.